edgefirewall.yaml

```diff
@@ -332,6 +359,64 @@ components:
           type: boolean
         waf_enabled:
           type: boolean
+
+    Behaviors:
+      properties:
+        name:
+          type: string
+          enum: [deny, drop, set_rate_limit, set_waf_ruleset, run_function, tag_event]
+      
+    SSLVerificationStatusOperators:
+      type: string
+      enum: [is_equal, is_not_equal]
+      
+    SSLVerificationStatusArguments:
+      type: string
+      enum: [SUCCESS, CERTIFICATE_VERIFICATION_ERROR, MISSING_CLIENT_CERTIFICATE]
+      
+    SSLVerificationStatusCriteria:
+      type: object
+      properties:
+        operator:
+          $ref: '#/components/schemas/SSLVerificationStatusOperators'
+        argument:
+          $ref: '#/components/schemas/SSLVerificationStatusArguments'
```

Comment:
**todo:** add the properties `variable` and `conditional`

`variable` = `ssl_verification_status`
`conditional` = `and` | `if`

Reply:
done.